Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -1383,7 +1462,9 @@ - (void)setActivityTitleWithText:(NSString *)text
   [linearView sizeToFit];
   
   self.logInSignOutCell.textLabel.text = nil;
-  [self.logInSignOutCell.contentView addSubview:linearView];
+  if (![self.logInSignOutCell.contentView viewWithTag:1]) {
```

Comment:
I don't love using `viewWithTag:` like this because it discards type/nullability information and involves magic constants. While my preference would just be to declare a new instance variable, I'd settle for defining an integer constant for this tag so we're not passing around `1` everywhere.